### PR TITLE
[formal] cleanup fpv_all related issues

### PR DESCRIPTION
**Pull request by cindychip**
Add env variable for 1."assumption and assertion checks"
                                 2. "coverage"
Reduce the parallel engine number
Add back prim_alert_rxtx_async_fpv core file

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/formal/fpv.tcl [hunk 1]
```diff
@@ -142,14 +143,15 @@ if {$env(FPV_TOP) == "top_earlgrey"} {
...
-check_assumptions -dead_end
+# run once to check if assertions and assumptions have any conflict
+# check_assumptions -conflict
+# check_assumptions -live
+# check_assumptions -dead_end
 
 #-------------------------------------------------------------------------
 # configure proofgrid
 #-------------------------------------------------------------------------
-set_proofgrid_per_engine_max_local_jobs 16
```
Comment (cindychip) [inline hw/formal/fpv.tcl]: I get some suggestions from other teams for using too many engines, and was recommended to reduce to 2.

**Comment by eunchan** [comment]
Can we use environment variable to run check only when that environment is defined?

**Comment by cindychip** [comment]
Thanks for the suggestion @eunchan.
Do you mean adding the environment variable for here:
```
# run once to check if assertions and assumptions have any conflict
# check_assumptions -conflict
# check_assumptions -live
# check_assumptions -dead_end
```

**Comment by eunchan** [comment]
Right.

**Comment by cindychip** [comment]
Thanks Eunchan, updated.

**Comment by eunchan** [comment]
LGTM :) Thanks. Waiting other reviewers approvals.

**Pull request merged**